### PR TITLE
allow usage of a custom GitLab URL

### PR DESCRIPTION
**Pull request by ahanselka**
This PR will add the ability to set a custom GitLab instance in `_config.yml` if you happen to be using Pages in a self-hosted environment.

This is pretty much my first contribution to a Ruby gem, so I welcome any feedback on making it better.

### CODE REVIEW

File: CHANGELOG.markdown [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.2.1
```
Comment (JustinAiken) [inline CHANGELOG.markdown]: - Usually when doing a PR to a gem/library, just put something like `NEXT`, and leave it to the maintainers to bump the version..
- Also to follow semvar, this is a new "feature", so I probably should bump it to `0.3.0`... but it's such a small feature I'll .z release it

So no worries this time, just a note for the next PRs you send 😄

Comment (ahanselka) [reply CHANGELOG.markdown]: Awesome, good to know. Thank you for the feedback!

**Comment by ahanselka** [comment]
I have used it with my own instance and with the commit I just added, it worked.

**Comment by JustinAiken** [comment]
Pushed out: https://github.com/JustinAiken/jekyll-gitlab-letsencrypt/releases/tag/v0.2.1

**Pull request merged**